lib/pdk/generators/module.rb

```diff
@@ -71,7 +71,10 @@ def self.invoke(opts = {})
         PDK.answers.update!('template-url' => template_url)
 
         begin
-          FileUtils.mv(temp_target_dir, target_dir)
+          if FileUtils.mv(temp_target_dir, target_dir)
+            PDK.logger.info(_('Module \'%{name}\' generated at path \'%{path}\'.') % { name: opts[:name], path: target_dir })
+            PDK.logger.info(_('In your new module directory, add classes with the \'pdk new class\' command.'))
+          end
```

Comment:
Is it possible for `FileUtils.mv` to fail without raising an exception?

Reply:
Not that I've seen

Reply:
Sorry, I should have clarified the point of my question which was, why do we need that conditional around the log lines?